core-web/libs/edit-content/src/lib/components/dot-edit-content-sidebar/components/dot-edit-content-sidebar-information/dot-edit-content-sidebar-information.component.html

```diff
@@ -31,12 +31,8 @@
                 <span class="sidebar-card__title">{{ 'Created' | dm }}</span>
                 <span
                     class="sidebar-card__subtitle"
-                    [tooltipPosition]="'left'"
-                    [pTooltip]="
-                        !contentlet?.creationDate
-                            ? ('edit.content.sidebar.information.no.created.yet' | dm)
-                            : null
-                    ">
+                    [pTooltip]="contentlet?.ownerName"
+                    tooltipPosition="left">
```

Comment:
@fmontes mentioned that tooltips should be below the icon/button instead of to the left or right. Does this apply here?

Reply:
Changed to use `tooltipPosition="bottom"` all of them.